### ISSUE TITLE
containers: no route to login node

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#476]
* r6
fd7ea53
 M controller/rootimg/install/postscripts/cv_install_dockerize

**System**: amaster

The containers cannot ping login.vc-a so slurm gives errors. 
```
[root@login ~]# systemctl status slurm
slurm.service - LSB: slurm daemon management
   Loaded: loaded (/etc/rc.d/init.d/slurm)
   Active: active (running) since Fri 2015-10-09 08:33:51 UTC; 9s ago
  Process: 9688 ExecStart=/etc/rc.d/init.d/slurm start (code=exited, status=0/SUCCESS)
 Main PID: 9697 (slurmctld)
   CGroup: /system.slice/slurm.service
           └─9697 /usr/sbin/slurmctld

Oct 09 08:33:50 login systemd[1]: Starting LSB: slurm daemon management...
Oct 09 08:33:51 login slurm[9688]: starting slurmctld: [  OK  ]
Oct 09 08:33:51 login systemd[1]: PID file /var/run/slurmctld.pid not readable (yet?) after start.
Oct 09 08:33:51 login systemd[1]: Started LSB: slurm daemon management.
[root@login ~]# sinfo
PARTITION  AVAIL  TIMELIMIT  NODES  STATE NODELIST
container*    up   infinite      2   unk* c[001-002]
[root@login ~]# systemctl status munge
munge.service - MUNGE authentication service
   Loaded: loaded (/usr/lib/systemd/system/munge.service; enabled)
  Drop-In: /etc/systemd/system/munge.service.d
           └─customexec.conf, remote-fs.conf
   Active: active (running) since Thu 2015-10-08 12:58:05 UTC; 19h ago
     Docs: man:munged(8)
 Main PID: 12263 (munged)
   CGroup: /system.slice/munge.service
           └─12263 /usr/sbin/munged --key-file /cluster/etc/munge/munge.key

Oct 08 12:58:05 login systemd[1]: Starting MUNGE authentication service...
Oct 08 12:58:05 login systemd[1]: Started MUNGE authentication service.
[root@login ~]# view /var/log/s
secure   slurm/   spooler  
[root@login ~]# view /var/log/slurm/slurmctld.log 
[root@login ~]# ls
[root@login ~]# sinfo
PARTITION  AVAIL  TIMELIMIT  NODES  STATE NODELIST
container*    up   infinite      2   unk* c[001-002]
[root@login ~]# exit
logout
Connection to login.vc-a closed.
[root@controller(unknown*) ~]# ssh node001
Last login: Thu Oct  1 17:07:36 2015 from controller-2.cluster
[root@node001 ~]# docker ps
CONTAINER ID        IMAGE                            COMMAND                CREATED             STATUS              PORTS               NAMES
f4ae1fea2e0a        controller:5050/trinity:latest   "/usr/bin/supervisor   3 minutes ago       Up 3 minutes                            trinity             
[root@node001 ~]# docker-enter trinity
[root@c001 /]# hostname
c001.vc-a
[root@c001 /]# sinfo
sinfo: error: Unable to resolve "login": Host name lookup failure
sinfo: error: Unable to establish control machine address
slurm_load_partitions: Resource temporarily unavailable
[root@c001 /]# host login
;; connection timed out; trying next origin
^C[root@c001 /]# cat /etc/resolv.conf 
; generated by /usr/sbin/dhclient-script
search cluster. vc-a.
nameserver 10.141.255.254
[root@c001 /]# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
8: eth1@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc noqueue state UNKNOWN 
    link/ether 36:f0:27:cf:d1:f7 brd ff:ff:ff:ff:ff:ff
    inet 172.16.0.1/16 scope global eth1
       valid_lft forever preferred_lft forever
    inet6 fe80::34f0:27ff:fecf:d1f7/64 scope link 
       valid_lft forever preferred_lft forever
9: ph3549eth1@if2: <BROADCAST,MULTICAST> mtu 9000 qdisc noop state DOWN 
    link/ether 5a:be:6c:fb:78:e8 brd ff:ff:ff:ff:ff:ff
[root@c001 /]# ping 10.141.255.254
PING 10.141.255.254 (10.141.255.254) 56(84) bytes of data.
From 172.16.0.1 icmp_seq=1 Destination Host Unreachable
From 172.16.0.1 icmp_seq=2 Destination Host Unreachable
From 172.16.0.1 icmp_seq=3 Destination Host Unreachable
From 172.16.0.1 icmp_seq=4 Destination Host Unreachable
^C
--- 10.141.255.254 ping statistics ---
4 packets transmitted, 0 received, +4 errors, 100% packet loss, time 2999ms
pipe 4
[root@c001 /]# 1444379965793	addons.update-checker	WARN	Update manifest for {972ce4c6-7e08-4474-a285-3208198ce6fd} did not contain an updates property

[root@c001 /]# exit
exit
[root@node001 ~]# exit
logout
Connection to node001 closed.
[root@controller(unknown*) ~]# route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         192.168.1.254   0.0.0.0         UG    0      0        0 enp2s0f0
10.141.0.0      0.0.0.0         255.255.0.0     U     0      0        0 br100
10.146.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
10.148.0.0      0.0.0.0         255.255.0.0     U     0      0        0 br100
link-local      0.0.0.0         255.255.0.0     U     1002   0        0 enp2s0f0
192.168.1.0     0.0.0.0         255.255.255.0   U     0      0        0 enp2s0f0
192.168.32.0    0.0.0.0         255.255.255.0   U     0      0        0 br100
```

**Comment by abhimukh** [comment]
Should be fixed by e2237532e49140a5502ede30012c1d0bd5c1c4a9

**Comment by themisa** [comment]
**Version:** r6 fd7ea53
**Environment:** tmaster

```
[root@controller(unknown*) clusterbats]# route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         192.168.1.254   0.0.0.0         UG    0      0        0 eno1
10.141.0.0      0.0.0.0         255.255.0.0     U     0      0        0 eno2
10.146.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
10.148.0.0      0.0.0.0         255.255.0.0     U     0      0        0 eno2
link-local      0.0.0.0         255.255.0.0     U     1002   0        0 eno1
link-local      0.0.0.0         255.255.0.0     U     1003   0        0 eno2
172.16.0.0      0.0.0.0         255.240.0.0     U     0      0        0 eno2
192.168.1.0     0.0.0.0         255.255.255.0   U     0      0        0 eno1
192.168.32.0    0.0.0.0         255.255.255.0   U     0      0        0 br100
```
